manifests/d/Docker/DockerDesktop/4.0.1/Docker.DockerDesktop.installer.yaml

```diff
@@ -0,0 +1,24 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+PackageIdentifier: Docker.DockerDesktop
+PackageVersion: 4.0.1
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: exe
+  Scope: machine
+  InstallerUrl: https://desktop.docker.com/win/main/amd64/Docker%20Desktop%20Installer.exe
```

Comment:
Thanks @soroshsabz 
Unfortunately with the 4.1.0 release of Docker Desktop your 4.0.1 package got automatically removed.
This is because of the `InstallerUrl` pointing to the always latest binary. 
To have stability another url with the build number is required as it's done in the past. For 4.0.1 it's build number 68347.
Are you planning to provide new packages in the future? I'm thinking of fixing our CI to provide a new winget package whenever we do a new release here at Docker.

Reply:
@StefanScherer Thanks for pointing that, I correct this and make another PR

Reply:
@StefanScherer please see #29147